debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleValueConverters.java

```diff
@@ -688,6 +689,17 @@ protected Object convertTimestampWithZone(Column column, Field fieldDefn, Object
         });
     }
 
+    private static Object convertToZonedDateTime(String dateText) {
+        Object data;
+        if (dateText.trim().startsWith("-")) {
+            data = ZonedDateTime.from(TIMESTAMP_TZ_FORMATTER.parse(dateText.trim().substring(1, dateText.length()))).with(ChronoField.ERA, 0);
+        }
+        else {
+            data = ZonedDateTime.from(TIMESTAMP_TZ_FORMATTER.parse(dateText.trim()));
+        }
+        return data;
+    }
```

Comment:
I'm not a fan of the repeated use of `trim()` here and in the caller, `convertTimestampWithZone`.  Can we unify this so that the text is already _trimmed_ as an input argument to this method?